hack/update-codecgen.sh

```diff
@@ -143,7 +143,7 @@ for current in "${index[@]}"; do
   ###echo "DBG: running ${CODECGEN} -d 1234 -o ${base_generated_file} ${base_file}"
   ${CODECGEN} -d 1234 -o "${base_generated_file}" "${base_file}"
   # Add boilerplate at the beginning of the generated file.
-  sed 's/YEAR/2015/' "${initial_dir}/hack/boilerplate/boilerplate.go.txt" > "${base_generated_file}.tmp"
+  sed 's/YEAR/2016/' "${initial_dir}/hack/boilerplate/boilerplate.go.txt" > "${base_generated_file}.tmp"
```

Comment:
I can do that, but on Jan 1 it will make verify-codecgen and verify-generated-protobuf instantly fail at head until someone checks in the fix.

I can't decide which is the greater evil: the known timebomb I'd create or the sysiphean toil of searching and replacing.

WDYT?